packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.html

```diff
@@ -1,212 +1,243 @@
+                        data-values="$ctrl.blockIpOffers"
+                        data-on-change="$ctrl.onSelectedOfferChange('IPv4Block')"
+                        data-variant="secondary"
+                        data-required
                     >
-                    </option>
-                </select>
-                <p
-                    data-ng-if="!ctrl.ipOffers.length"
-                    data-translate="ip_order_quota_full"
-                ></p>
+                        <oui-select-picker-description>
+                            <strong>
+                                <h4
+                                    class="text-center"
+                                    data-translate="ip_agora_block_ip_title"
+                                ></h4>
+                                <p
+                                    class="text-center"
+                                    data-translate="ipv4Block_price_label"
+                                ></p>
+                            </strong>
+                            <p
+                                class="text-center"
+                                data-translate="ipv4Block_description"
+                            ></p>
+                        </oui-select-picker-description>
+                    </oui-select-picker>
+                </div>
             </div>
 
-            <div data-ng-show="ctrl.model.params.selectedOffer">
+            <div
+                data-ng-show="$ctrl.model.params.selectedOffer"
+                data-ng-if="!$ctrl.loading.ipOffers"
+            >
                 <div
-                    class="form-group"
-                    data-ng-show="!ctrl.model.params.selectedOffer.isIpBlockOffer"
+                    data-ng-show="!$ctrl.model.params.selectedOffer.isIpBlockOffer"
                 >
-                    <label
-                        for="quantity"
-                        class="control-label"
-                        data-translate="ip_agora_table_header_quantity"
-                    >
-                    </label>
-                    <select
-                        id="quantity"
-                        class="form-control"
-                        data-ng-model="ctrl.model.params.selectedQuantity"
-                        data-ng-options="quantity for quantity in ctrl.model.params.selectedOffer.quantities track by quantity"
-                    >
-                        <option
-                            value=""
-                            data-ng-disabled="true"
-                            data-translate="global_select"
-                        >
-                        </option>
-                    </select>
+                    <div class="row">
+                        <strong>
+                            <p
+                                data-translate="{{ 'ip_agora_select_quantity_title' | translate }}"
+                            ></p>
+                        </strong>
```

Comment:
not tataly sure but i think we have to use theme like this:
```suggestion
                        <p>
                            <strong
                                data-translate="{{ 'ip_agora_select_quantity_title' | translate }}"
                            ></strong>
                        </p>
```